azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -15,7 +88,118 @@ def _get_configurations(**kwargs) -> Dict[str, ConfigurationValue]:
+    _default_tracing_export_interval_ms(configurations)
+    _default_instrumentation_config(configurations)
+
+    # TODO: remove when validation added to BLRP
+    if configurations[LOGGING_EXPORT_INTERVAL_MS_ARG] <= 0:
+        raise ValueError("%s must be positive." % LOGGING_EXPORT_INTERVAL_MS_ARG)
+
     return configurations
 
 
-# TODO: Add env var configuration
+def _default_exclude_instrumentations(configurations):
+    if EXCLUDE_INSTRUMENTATIONS_ARG not in configurations:
+        default = []
+        if EXCLUDE_INSTRUMENTATIONS_ENV_VAR in environ:
+            default = loads(environ[EXCLUDE_INSTRUMENTATIONS_ENV_VAR])
+        configurations[EXCLUDE_INSTRUMENTATIONS_ARG] = default
+
+
+def _default_disable_logging(configurations):
+    if DISABLE_LOGGING_ARG not in configurations:
+        default = False
+        if DISABLE_LOGGING_ENV_VAR in environ:
+            default = bool(environ[DISABLE_LOGGING_ENV_VAR])
+        configurations[DISABLE_LOGGING_ARG] = default
+
+
+def _default_disable_metrics(configurations):
+    if DISABLE_METRICS_ARG not in configurations:
+        default = False
+        if DISABLE_METRICS_ENV_VAR in environ:
+            default = bool(environ[DISABLE_METRICS_ENV_VAR])
+        configurations[DISABLE_METRICS_ARG] = default
+
+
+def _default_disable_tracing(configurations):
+    if DISABLE_TRACING_ARG not in configurations:
+        default = False
+        if DISABLE_TRACING_ENV_VAR in environ:
+            default = bool(environ[DISABLE_TRACING_ENV_VAR])
+        configurations[DISABLE_TRACING_ARG] = default
+
+
+def _default_logging_level(configurations):
+    if LOGGING_LEVEL_ARG not in configurations:
+        default = NOTSET
+        if LOGGING_LEVEL_ENV_VAR in environ:
+            default = int(environ[LOGGING_LEVEL_ENV_VAR])
+            # TODO: Match OTel env var usage when it is determined.
+        configurations[LOGGING_LEVEL_ARG] = default
+
+
+def _default_logger_name(configurations):
+    if LOGGER_NAME_ARG not in configurations:
+        default = ""
+        if LOGGER_NAME_ENV_VAR in environ:
+            default = environ[LOGGER_NAME_ENV_VAR]
+        configurations[LOGGER_NAME_ARG] = default
+
+
+def _default_logging_export_interval_ms(configurations):
+    if LOGGING_EXPORT_INTERVAL_MS_ARG not in configurations:
+        default = 5000
+        if LOGGING_EXPORT_INTERVAL_MS_ENV_VAR in environ:
+            default = int(environ[LOGGING_EXPORT_INTERVAL_MS_ENV_VAR])
+        configurations[LOGGING_EXPORT_INTERVAL_MS_ARG] = default
+
+
+def _default_metric_readers(configurations):
+    if METRIC_READERS_ARG not in configurations:
+        default = []
+        if METRIC_READERS_ENV_VAR in environ:
+            default = loads(environ[METRIC_READERS_ENV_VAR])
+        configurations[METRIC_READERS_ARG] = default
+
+
+def _default_views(configurations):
+    if VIEWS_ARG not in configurations:
+        #TODO tuple or list
+        default = []
```

Comment:
Was previously (). List makes more sense to me and matches the tests. @lzchen was there a reason for "()"

Reply:
OpenTelemetry [default](https://github.com/open-telemetry/opentelemetry-python/blob/b6a1b22fa65f41bdefb01d64b76e5e793d039f6d/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/__init__.py#L346)

Reply:
Changed